scripts/soroban-settings/write_upgrade_bytes/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 use soroban_sdk::{contract, contractimpl, Env, Bytes, BytesN};
 
 pub(crate) const BUMP_AMOUNT: u32 = 518400; // 30 days
+pub(crate) const BALANCE_LIFETIME_THRESHOLD: u32 = BUMP_AMOUNT;
```

Comment:
Why `BALANCE_`? Seems like you can just do `bump(BUMP_AMOUNT, BUMP_AMOUNT)` (we don't anticipate this to be called more than just once, so no need to make the lower bound different).

Reply:
Nice catch! This is a copy/paste from an example in env. How about naming it as `EXPIRATION_LOW_WATERMARK`?

Reply:
Yeah, that's ok as well.